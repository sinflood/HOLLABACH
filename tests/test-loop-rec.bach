inst Banjo{
loop 4{
[C3/2]
loop 2{
[B3/2]
[A3/2]
}
}
}
