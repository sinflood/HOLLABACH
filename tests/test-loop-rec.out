Timing Resolution (pulses per quarter note)
4

Instrument,105,Banjo,

Tick,Note(0-127),Velocity(0-127),Length,
0,48,80,8
16,59,80,8
32,57,80,8
48,59,80,8
64,57,80,8
80,48,80,8
96,59,80,8
112,57,80,8
128,59,80,8
144,57,80,8
160,48,80,8
176,59,80,8
192,57,80,8
208,59,80,8
224,57,80,8
240,48,80,8
256,59,80,8
272,57,80,8
288,59,80,8
304,57,80,8
